test/CreateTest.php

```diff
@@ -92,35 +101,32 @@ public function test_05_flush_construct() {
 			$c->flush_schema();
 		}
 
-		/**
-		 * @expectedException TarantoolClientError
-		 * @expectedExceptionMessage Incorrect password supplied for user
-		 */
 		public function test_06_bad_credentials() {
+				$this->expectException(TarantoolClientError::class);
```

Comment:
I'm not sure if this is a good name for an exception. According to the [PHP error class hierarchy](https://3v4l.org/sDMsv), "errors" refer to internal PHP errors, and "exceptions" refer to userspace errors. I would rename this class to avoid confusion.

Reply:
Thanks for notice, but it surely should not be changed within this PR (it is incompatible change).

Don't sure whether it worth to file an issue. I don't a PHP developer, so I have very small context here. If you think that the breaking change worth it, can you, please, file an issue?

Reply:
Ah, indeed, I missed that this exception had existed for a long time already.